crossdock/server/endtoend.py

```diff
@@ -99,7 +101,7 @@ def generate_traces(self, request, response_writer):
         tracer = self.tracers[sampler_type]
         for _ in range(req.get('count', 0)):
             span = tracer.start_span(req['operation'])
-            for k, v in req.get('tags', {}).iteritems():
+            for k, v in req.get('tags', {}).items():
```

Comment:
Yeah, I agree. I was coming from an assumption that there will be not so many tags per span and spans will be sampled (i.e. 1 span per 100 or 1000 operation runs) so the performance was not a huge concern from my point of view.

To keep performance on the same level, something like this helper function should probably work better:
```
def iteritems(d):
  if sys.version_info[0] == 2:
    return d.iteritems()
  else:
    return iter(d.items())
```

Well, it's already a part of the `six` [package](https://github.com/JioCloud/python-six/blob/master/six.py#L573) – I'll go ahead and change that.